src/java-opentimelineio/src/main/java/io/opentimeline/opentimelineio/Any.java

```diff
@@ -0,0 +1,169 @@
+package io.opentimeline.opentimelineio;
+
+import io.opentimeline.OTIONative;
+import io.opentimeline.opentime.RationalTime;
+import io.opentimeline.opentime.TimeRange;
+import io.opentimeline.opentime.TimeTransform;
+
+public class Any extends OTIONative {
+
+    public Any(long nativeHandle) {
+        this.nativeHandle = nativeHandle;
+    }
+
+    public Any(boolean b) {
+        this.initBool(b);
+    }
+
+    public Any(int i) {
+        this.initInt(i);
+    }
+
+    public Any(double d) {
+        this.initDouble(d);
+    }
+
+    public Any(String string) {
+        this.initString(string);
+    }
+
+    public Any(RationalTime rationalTime) {
```

Comment:
Might be less code if you a generic Any<T> or even just take a generic argument into constructor and initialize using instanceof checks.